middleware/cached.js

```diff
@@ -1,10 +1,16 @@
 'use strict';
-const config = require('config');
+const ms = require('ms');
 const csurf = require('csurf');
 const vary = require('vary');
 const cacheControl = require('express-cache-controller');
 
-const defaultMaxAge = config.get('viewCacheExpiration');
+const toSeconds = naturalTime => {
+    const parsedMilliseconds = typeof naturalTime === 'string' ? ms(naturalTime) : naturalTime;
```

Comment:
You can probably do this with `moment` already, like `moment.duration(4, 'weeks').asMilliseconds()` etc? But perhaps doesn't have the natural time support.

Reply:
Doesn't seem to have the shorthand syntax.